common/pulp_docker/common/tarutils.py

```diff
@@ -114,3 +114,24 @@ def get_youngest_children(metadata):
                 pass
 
     return list(image_ids)
+
+
+def get_image_manifest(tarfile_path):
+    """
+    Given a path to a tarfile, which itself is the product of "skopeo copy"
+    this returns the decoded manifest.json file if it exists.
```

Comment:
i am not very familiar with skopeo tool, but should not it be a requirement to contain the manifest.json file in the tarfile? otherwise the image should be invalid/corrupted?